Makefile.autosetup

```diff
@@ -58,20 +58,23 @@ VPATH=		@VPATH@
 
 ALL_FILES!=	(cd $(SRCDIR) && git ls-files 2>/dev/null) || true
 
+all:
```

Comment:
Is this needed?

Reply:
Yes, it makes sure that the default target is `all`.
The full line, later, adds the actual details:
```make
all: $(BINFILES) $(LIBBINFILES) $(ALL_TARGETS)
```